django_celery_results/models.py

```diff
@@ -95,12 +95,18 @@ class Meta:
         verbose_name = _('task result')
         verbose_name_plural = _('task results')
 
+        # Explicit names to solve https://code.djangoproject.com/ticket/33483
```

Comment:
since the ticket is won't fix, I am reluctant to include this right now. Instead, can't we define index name in the model meta to generate nicer/shorter names?

Reply:
Not sure if I understand you correctly. I am defining index names in model meta.

I can change the names to be prettier, but then it would mean a migration that drops and re-creates all indexes (there's no `RenameIndex` migration in Django). For users who have lots of data, that can be disruptive. But if you think it's worth it, I can do it.

Reply:
Actually a 3rd option is to use `db_index=True` in field definitions, instead of defining `Meta.indexes`. Indexes created that way are database-engine-agnostic.

![image](https://user-images.githubusercontent.com/137616/152337878-a813b117-99c8-48ca-b913-314fd033ba89.png)

Reply:
These fields used to have db_index=True and were changed to the Meta, causing all sorts of index issues:
https://github.com/celery/django-celery-results/pull/161#discussion_r607473655